docs/faq.asciidoc

```diff
@@ -0,0 +1,44 @@
+[[faq]]
+== Frequently Asked Questions
+
+[float]
+[[faq-how-does-it-work]]
```

Comment:
I think a lot of these need the `[descrete]` tag

Reply:
There are some subtle differences, but for the majority of our use cases `[descrete]` == `[float]`

Reply:
Thanks @bmorelli25